string model_name # Name of the model to change
string new_model_urdf  # name of the urdf file for the new model
